frappe/public/js/frappe/views/reports/query_report.js

```diff
@@ -1383,7 +1383,16 @@ frappe.views.QueryReport = class QueryReport extends frappe.views.BaseList {
 			layout_direction: frappe.utils.is_rtl() ? "rtl" : "ltr",
 		});
 
-		frappe.render_pdf(html, print_settings);
+		let filter_values = [],
+			name_len = 0;
+		for (var key of Object.keys(applied_filters)) {
+			name_len = name_len + applied_filters[key].toString().length;
+			if (name_len > 200) break;
+			filter_values.push(applied_filters[key]);
+		}
+		print_settings.report_name = `${__(this.report_name)}_${filter_values.join("_")}.pdf`;
+		console.log(print_settings.report_name);
+		// frappe.render_pdf(html, print_settings);
```

Comment:
This doesn't seem intentional also why the render function is commented out?

Reply:
The render function cleared the console. So I had commented it out during testing the name length and forgot to revert :grimacing: